exams/pearson/constants.py

```diff
@@ -6,6 +6,23 @@
 
 PEARSON_FILE_TYPE_EAC = 'eac'
 PEARSON_FILE_TYPE_VCDC = 'vcdc'
+PEARSON_FILE_TYPE_EXAM = 'exam'
+PEARSON_FILE_TYPE_CAND = 'cand'
+PEARSON_FILE_TYPE_SURV = 'surv'
+PEARSON_FILE_TYPE_CMNT = 'cmnt'
+PEARSON_FILE_TYPE_SECT = 'sect'
+PEARSON_FILE_TYPE_RESP = 'resp'
+PEARSON_FILE_TYPE_ITEM = 'item'
+
+# these are files we intentionally skip, but treat as if we processed them
+PEARSON_INTENDED_SKIP_FILE_TYPES = (
+    PEARSON_FILE_TYPE_CAND,
+    PEARSON_FILE_TYPE_SURV,
+    PEARSON_FILE_TYPE_CMNT,
+    PEARSON_FILE_TYPE_SECT,
+    PEARSON_FILE_TYPE_ITEM,
+    PEARSON_FILE_TYPE_RESP,
+)
```

Comment:
Would be nice to namespace these and make a namedtuple (et al) called `PEARSON_FILE_TYPES`. We'd then get/set these values like `PEARSON_FILE_TYPES.CAND`, `PEARSON_FILE_TYPES.SURV` etc., but that can be done in a different PR